app/assets/javascripts/application.js

```diff
@@ -15,3 +15,13 @@
 //= require rails-ujs
 //= require turbolinks
 //= require_tree .
+
+$(document).ready(function() {
```

Comment:
phần này viết riêng ra file js khác nhé
file này chỉ đưa các config chung cho toàn project thôi